genny/auth/auth.go

```diff
@@ -74,14 +76,20 @@ func New(args []string) (*genny.Generator, error) {
 		gf, err = gogen.AddInsideBlock(
 			gf,
 			`if app == nil {`,
+			`s:= MockSender{}`,
 			`app.Use(SetCurrentUser)`,
 			`app.Use(Authorize)`,
+			`app.Use(SetupRecoverySender(s))`,
 			`app.GET("/users/new", UsersNew)`,
 			`app.POST("/users", UsersCreate)`,
 			`app.GET("/signin", AuthNew)`,
 			`app.POST("/signin", AuthCreate)`,
 			`app.DELETE("/signout", AuthDestroy)`,
-			`app.Middleware.Skip(Authorize, HomeHandler, UsersNew, UsersCreate, AuthNew, AuthCreate)`,
+			`app.GET("/recovery", UserRecovery)`,
+			`app.POST("/requestRecovery", UserRequestRecovery)`,
+			`app.GET("/recover", UserRecover)`,
+			`app.POST("/requestRecover", UserRequestRecover)`,
```

Comment:
I found the paths a bit confusing and too similar to eachother.
Maybe something like `recovery` -> `requestRecovery` -> `recover` -> `completeRecovery` would be a bit more clear, though I still don't like the `recover` very much, I just didn't find a better wording ^^

Reply:
sounds good!